apps/editor.planx.uk/src/pages/Preview/ApplicationViewer.tsx

```diff
@@ -0,0 +1,51 @@
+import Box from "@mui/material/Box";
+import Typography from "@mui/material/Typography";
+import { FeeBreakdown } from "@planx/components/Pay/Public/FeeBreakdown/FeeBreakdown";
+import SummaryListsBySections from "@planx/components/shared/Preview/SummaryList";
+import { PrintButton } from "components/PrintButton";
+import { useStore } from "pages/FlowEditor/lib/store";
+import React from "react";
+
+import StatusPage from "./StatusPage";
+
+const NoContentPage = () => (
+  <StatusPage
+    bannerHeading="Your form"
+    buttonText="Start a new form"
+  >
+    <Typography variant="body2">
+      No answers to display
+    </Typography>
+  </StatusPage>
+)
+
+const ApplicationViewer: React.FC = () => {
+  const [flow, passport, breadcrumbs, govUkPayment] = useStore(state => [state.flow, state.computePassport(), state.breadcrumbs, state.govUkPayment]);
+  const hasPaymentInfo = Boolean(govUkPayment);
```

Comment:
nit: I think this will fail to correctly catch ITP which does _not_ set `govUkPayment` key? 

Instead I think we just want to check breadcrumbs for presence of `Pay` component type (in _any_ mode including info-only)?